docs/docs/reference/configuration.md

```diff
@@ -496,4 +498,94 @@ Shutdown period of metric telemetry in milliseconds.
 
 * Example value: `10000`
 * Type: `optional`
-* Default value: `10000`
\ No newline at end of file
+* Default value: `10000`
+
+### Python Udfs
+
+#### `PYTHON_UDF_ENABLE`
+
+Enable/Disable using python udf.
+
+* Example value: `10000`
+* Type: `optional`
+* Default value: `10000`
+
+#### `PYTHON_UDF_CONFIG`
+
+All the configuration need to use python udf.
+
+These following variables need to be configured:
+
+##### `PYTHON_FILES`
+
+Defines the path of python udf files. Currently only support for `.py` and `.zip` data type. Comma (',') could be used as the separator to specify multiple files.
+
+* Example value: `/path/to/files.zip`
+* Type: `required`
+
+##### `PYTHON_ARCHIVES`
+
+Defines the path of files that used on the python udf. Only support for `.zip` data type. Comma (',') could be used as the separator to specify multiple archive files.
+'#' could be used as the separator of the archive file path and the target directory name.
+
+Example:
+* PYTHON_ARCHIVES=/path/to/data.zip
+
+  You should set the path name to `data.zip/data/sample.txt`
+  
+* PYTHON_ARCHIVES=/path/to/data.zip#data
+  
+  You should set the path name to `data/sample.txt` 
```

Comment:
This might get a little difficult to understand for someone with no context, lets rewrite it in more descriptive way